pkg/commands/artifact/run.go

```diff
@@ -350,6 +366,10 @@ func run(ctx context.Context, opt Option, artifactType ArtifactType) (err error)
 		if report, err = runner.ScanRepository(ctx, opt); err != nil {
 			return xerrors.Errorf("repository scan error: %w", err)
 		}
+	case cycloneDXArtifact:
+		if report, err = runner.ScanCycloneDX(ctx, opt); err != nil {
```

Comment:
```suggestion
		if report, err = runner.ScanSBOM(ctx, "cyclonedx", opt); err != nil {
```

"cyclonedx" should be constant though.